frontend/src/components/AddRecipeForm.js

```diff
@@ -11,6 +11,28 @@ const showAlert = () => {
 };
 
 const AddRecipeForm = () => {
+  const [newCategory, setNewCategory] = useState('');
+  const [error, setError] = useState('');
+  async function addCategory(value) {
+    const data = {
+      name: value,
+    };
+
+    console.log(data);
+    data.name !== ''
+      ? setCategoryList([...categoryList, data])
+      : setCategoryList([...categoryList]);
+    try {
+      await axios.post(`http://localhost:8081/api/categories`, data);
+      setError('Sikeres hozzáadás!');
```

Comment:
Akkor ne error legyen, ha a sikeres esetben is ezt töltjük föl, legyen status vagy message.

Reply:
Done.